### YamlMime:UniversalReference
ms.openlocfilehash: 5f5c3412206e4f0c34dbbbe6ab523d1f968819e1
ms.sourcegitcommit: 286f52ea38c9eff2ec9d4f8cabeb86f62fd9c406
ms.translationtype: MT
ms.contentlocale: ko-KR
ms.lasthandoff: 08/22/2018
ms.locfileid: "41687701"
items:
- uid: azure-iot-common.CloudToDeviceDetachedError
  name: CloudToDeviceDetachedError
  fullName: CloudToDeviceDetachedError
  children:
  - azure-iot-common.CloudToDeviceDetachedError.constructor
  - azure-iot-common.CloudToDeviceDetachedError.Error
  - azure-iot-common.CloudToDeviceDetachedError.innerError
  langs:
  - typeScript
  type: class
  summary: <span data-ttu-id="633f0-101">전송 수준에서 작업, 시작을 다시 시도 c2d 기능이 중지 될 때 발생 하는 데 오류가 발생 했습니다.</span><span class="sxs-lookup"><span data-stu-id="633f0-101">Error thrown when the c2d feature stopped working at the transport level, requiring the client to retry starting it.</span></span>
  extends:
    name: Error
  package: azure-iot-common
- uid: azure-iot-common.CloudToDeviceDetachedError.constructor
  name: CloudToDeviceDetachedError
  children: []
  type: constructor
  langs:
  - typeScript
  summary: ''
  syntax:
    content: 'new CloudToDeviceDetachedError(message?: string)'
    parameters:
    - id: message
      type:
      - string
      description: ''
      optional: true
  package: azure-iot-common
- uid: azure-iot-common.CloudToDeviceDetachedError.Error
  name: Error
  fullName: Error
  children: []
  langs:
  - typeScript
  type: property
  summary: ''
  syntax:
    content: 'static Error: ErrorConstructor'
    return:
      type:
      - ErrorConstructor
  package: azure-iot-common
- uid: azure-iot-common.CloudToDeviceDetachedError.innerError
  name: innerError
  fullName: innerError
  children: []
  langs:
  - typeScript
  type: property
  summary: ''
  syntax:
    content: 'innerError: Error'
    return:
      type:
      - Error
  package: azure-iot-common
